docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -0,0 +1,680 @@
+. Open the `rhoas_kafka.yml` file in your IDE.
+. In the top section of the new playbook, you define the group of hosts on which to run the playbook and its name. In the following example, the host is a local host and the connection is local because you are in a virtual environment. Copy the top section example and paste it into the `rhoas_kafka.yml` file.
++
+.Example top section for `rhoas_kafka` playbook
+[source,yaml]
+----
+- name: RHOAS kafka
+  hosts: localhost
+  gather_facts: false
+  connection: local
+  tasks:
+----
+
+. Next, you define the list of tasks. Copy the `create_kafka_` module example and paste it into the `rhoas_kafka.yml` file after the top section to provision a new Kafka instance.
+. Enter a name for the Kafka instance in the `name` field. You can specify configuration options such as `billing_cloud_account_id` and the `openshift_offline_token`. All other information for the instance is provided by the Kafka APIs. The default contents of this module in the OpenShift Application Services example playbook are shown below:
++
+.Example `create_kafka` module
+[source,yaml]
+----
+- name: Create kafka
+    rhoas.rhoas.create_kafka:
+      name: "kafka-name"
+      instance_type: "x1"
+      billing_model: "standard"
+      cloud_provider: "aws"
+      region: "us-east-1"
+      plan: "developer.x1"
+      billing_cloud_account_id: "123456789"
+      openshift_offline_token: "OPENSHIFT_OFFLINE_TOKEN"
+    register:
+      kafka_req_resp
+----
++
+When you run the `create_kafka_` module as part of the playbook at the end of this procedure, Ansible saves the output of that command in a variable in the `register` field. In the preceding example, Ansible will save the created Kafka instance as `kafka_req_resp`.
+
+. To connect your applications or services to a Kafka instance in {product-kafka}, you must first create a service account with credentials. Copy the `create_service_account` module and paste it into the `rhoas_kafka.yml` file to create a service account.
+. Enter values for the `name` and `short description` fields. When you run the `create_service_account` module as part of the playbook at the end of this procedure, Ansible populates the generated service account credentials in the `client_id` and `client_secret` fields in the terminal after it creates the service account.
++
+.Example `create_service_account` module
+[source,yaml]
+----
+- name: Create Service Account
+    create_service_account:
+      name: "service-account-name"
+      description: "This is a description of the service account"
+      openshift_offline_token: "OPENSHIFT_OFFLINE_TOKEN"
+    register:
+      srvce_acc_resp_obj
+----
+
+. Copy the contents of the `create_kafka_acl_binding` module and paste it into the `rhoas_kafka.yml` file to create an ACL for the service account with some default values, and bind that ACL to the Kafka instance and service account.
```

Comment:
Suggest: "To create Access Control List (ACL) permissions for the service account and bind that ACL to the Kafka instance, copy the `create_kafka_acl_binding` from the example playbook and paste it in your rhoas_kafka.yml` file."